### ISSUE TITLE
Make Wabbajack detect [No Delete] as well, instead of only [NoDelete].

### ISSUE THREAD

**Issue by Ullrrific** [wabbajack-tools/wabbajack#2351]
Had an issue where half of my mods were removed on an update. I was ignorant about Wabbajack not detecting NoDelete with a space between the two words. It's a small request, but it would likely save some people in the future from experiencing the problem I have to deal with now. 

When people relay this info to new users, they sometimes add a space in the middle of NoDelete which is an easy mistake to make. It's best to just remove such an easy typo.

tl;dr: Make Wabbajack ignore folders starting with [NoDelete] **_AND_** [No Delete].

**Comment by EzioTheDeadPoet** [comment]
I would consider a better solution to add the information about the feature simply to the Wiki.
But we can look into it.
